src/zcl_abapgit_repo.clas.abap

```diff
@@ -500,6 +505,16 @@ CLASS ZCL_ABAPGIT_REPO IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD get_or_create_log.
```

Comment:
Can this be merged into get_log ? 

P.S. @larshp could you pls have a look on this too. I didn't see into repo code for quite a while . Also I rememeber there was an intention to remove log from repo at all.